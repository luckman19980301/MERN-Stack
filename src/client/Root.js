import React from 'react';
import { Provider } from 'react-redux';

import store from './redux';

class Root extends React.Component {
  render() {
    const { children } = this.props;
    return <Provider store={store}>{children}</Provider>;
  }
}

export default Root;
