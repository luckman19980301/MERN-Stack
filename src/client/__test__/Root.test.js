import React from 'react';
import { shallow } from 'enzyme';
import Root from '../Root';

describe('Root component', () => {
  const wrapper = shallow(<Root />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
