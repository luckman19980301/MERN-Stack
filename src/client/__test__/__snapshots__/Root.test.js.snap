// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root component renders correctly 1`] = `ShallowWrapper {}`;
